app/models/service.rb

```diff
@@ -5,6 +5,8 @@ class Service < ActiveRecord::Base
   has_many :discontinued_services
 
   validates_presence_of :recorded_by_educator_id, :student_id, :service_type_id, :recorded_at, :date_started
+  validates :provided_by_educator_name, allow_nil: true,
+            format: { with: /\w+. \w+/ }
```

Comment:
An excellent point! It's weird that once I took the message away, Rails never complained that everything had the wrong form. Guess it's a necessary part of the validate statement.